airflow/utils/log/file_task_handler.py

```diff
@@ -253,10 +309,10 @@ def _read(self, ti: TaskInstance, try_number: int, metadata: dict[str, Any] | No
                     )
                 # Check if the resource was properly fetched
                 response.raise_for_status()
-
                 log += "\n" + response.text
             except Exception as e:
-                log += f"*** Failed to fetch log file from worker. {str(e)}\n"
+                service_to_fetch_from = LogType.TRIGGER if log_type == LogType.TRIGGER else LogType.WORKER
+                log += f"*** Failed to fetch log file from {service_to_fetch_from}. {str(e)}\n"
```

Comment:
```suggestion
                log += f"*** Failed to fetch log file from {log_type}. {str(e)}\n"
```

I think

Reply:
Or maybe this
```suggestion
                log += f"*** Failed to fetch log file from {log_type or LogType.WORKER}. {str(e)}\n"
```

Reply:
Or maybe we set the default to `log_type = LogType.WORKER` instead of None?

Reply:
> Or maybe we set the default to `log_type = LogType.WORKER` instead of None?

probably best